app/assets/stylesheets/pageflow/admin/embedded_index_table.scss

```diff
@@ -1,6 +1,10 @@
+// since activeadmin 2.5.0 depends on ruby 2.4
+// this is a temporary fix to use both ruby 2.3 and ruby 2.4 
```

Comment:
This is more about compatibility with ActiveAdmin < 2.5 (or whatever version introduced the icon urls) than about Ruby version compatibility.